modules/persistent-storage-csi-olm-driver-uninstall.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
+
+[id="persistent-storage-csi-olm-driver-uninstall_{context}"]
+= Uninstalling the {FeatureName} CSI driver
+
+.Prerequisites
+* Access to the {product-title} web console.
```

Comment:
All applications that use EFS PVs must be shut down. Maybe even a stronger warning that all EFS PVs will be inaccessible after driver uninstallation would be appropriate.